akka-docs/src/test/java/jdocs/actor/DependencyInjectionDocTest.java

```diff
@@ -50,8 +48,9 @@ public static void beforeClass() {
   }
 
   @AfterClass
-  public static void afterClass() throws Exception {
-    Await.ready(system.terminate(), Duration.create("5 seconds"));
+  public static void afterClass() {
+    //Await.ready(system.terminate(), Duration.create("5 seconds"));
```

Comment:
Seems good, remove the commented-out line?

Reply:
Done